projects/js-packages/charts/src/components/line-chart/line-chart.tsx

```diff
@@ -120,6 +120,16 @@ const getCurveType = ( type?: CurveType, smoothing?: boolean ) => {
 	}
 };
 
+const LINE_CHART_DEFAULTS = {
```

Comment:
Sorry to be a pain, but I don't see this pattern anywhere and I don't think the approach improves readability or maintainability, so I'm pretty keen to keep the defaults inlined as they were. Happy to be persuaded; otherwise let's revert all those changes.

Reply:
Ok, reverted and removed 😄